backfill/2022-01-13-regrets-reporter-ucs/02-gcp-ingestion-beam-decoder.bash

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -exo pipefail
+
+# This script assumes it's being run from the ingestion-beam directory of the gcp-ingestion repo
+# with the code modified to convert any events[*].timestamp float values into integers.
+
+PROJECT="moz-fx-data-backfill-10"
+JOB_NAME="regrets-reporter-ucs-backfill"
+: "${START_DATE:=2021-11-17}"
+: "${END_DATE:=2022-01-09}"
+
+mvn \
+  clean \
+  compile \
+  exec:java -Dexec.mainClass=com.mozilla.telemetry.Decoder -Dexec.args="\
+      --runner=Dataflow \
+      --jobName=$JOB_NAME \
+      --project=$PROJECT \
+      --geoCityDatabase=gs://moz-fx-data-prod-geoip/GeoIP2-City/20220107/GeoIP2-City.mmdb \
+      --geoIspDatabase=gs://moz-fx-data-prod-geoip/GeoIP2-ISP/20220107/GeoIP2-ISP.mmdb \
+      --geoCityFilter=gs://moz-fx-data-prod-dataflow-templates/cities15000.txt \
+      --schemasLocation=gs://moz-fx-data-prod-dataflow/schemas/202201070303_0c984084.tar.gz \
+      --inputType=bigquery_table \
+      --input=\"$PROJECT:payload_bytes_error.structured\" \
```

Comment:
I was thinking for future backfills it might make sense to reverse these; have `input` be `backfill` for all jobs and `output` be the pipeline family's error table (in the backfill project). Thoughts?

Reply:
I agree that would make more sense.

Another idea would be putting the source table(s) in a dedicated dataset (e.g. `input`), which would be especially helpful if there are multiple input tables involved.  If that were used in this case, I imagine the source table would have been named like `input.payload_bytes_error_structured`.